src/site/content/en/metrics/custom-metrics/index.md

```diff
@@ -277,6 +270,69 @@ try {
   attribute will be ignored.
 {% endAside %}
 
+### Event Timing API
+
+The [First Input Delay (FID)](/fid/) metric measures the time from when a user
+first interacts with a page to the time when the browser is actually able to
+begin processing event handlers in response to that interaction. However, in
+some cases it may also be useful to measure the event processing time itself as
+well as the time until the next frame can be rendered.
+
+This is possible with the [Event Timing
```

Comment:
Drive-by: should we mention when it's shipping, as it's not on Stable yet? M85.

Reply:
This one is tricky because the `first-input` part of the Event Timing API does exist, and it's technically what this section is referring to. (But as you point out it won't work for general event timing use cases until M85.)